extrair_historico_betterstack.py

```
@@ -0,0 +1,134 @@
+END_DATE = "2025-12-17"
+
+BASE_URL_V2 = "https://uptime.betterstack.com/api/v2"
+BASE_URL_V3 = "https://uptime.betterstack.com/api/v3"
+
+def fetch_response_times(token, monitor_id, from_date, to_date):
+    url = f"{BASE_URL_V2}/monitors/{monitor_id}/response-times"
+    headers = {"Authorization": f"Bearer {token}"}
+    params = {"from": from_date, "to": to_date}
+    
+    print(f"Fetching response times from {from_date} to {to_date}...")
+    response = requests.get(url, headers=headers, params=params)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)